### PR TITLE
Handlebars updates

### PR DESCRIPTION
**Pull request by danshoup**
handlebars updates: updated newEvents page, and the event and homepage.

### CODE REVIEW

File: public/js/datepicker.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
+
+
+var victorApp=angular.module('myapp',[])
```
Comment (ddaghlas) [inline public/js/datepicker.js:3]: I'm thinking this might need to changed from var to const or let

File: public/js/datepicker.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+
+
+victorApp.controller('main', ['$scope', function ($scope) {
```
Comment (ddaghlas) [inline public/js/datepicker.js:7]: Trying to figure out what victorApp is... Can you explain its functionality?

File: views/newEvent.handlebars [hunk 3]
```diff
@@ -1,112 +1,142 @@
...
+            <div class="form-group">
+              <label for="inputAddress2">Address 2</label>
+              <input type="text" class="form-control" id="inputAddress2" placeholder="Apartment, studio, or floor">
+            </div>
+            <div class="form-row">
+              <div class="form-group col-md-6">
+                <label for="inputCity">City</label>
+                <input type="text" class="form-control" id="inputCity">
+              </div>
+              <div class="form-group col-md-4">
```
Comment (ddaghlas) [inline views/newEvent.handlebars:30]: Lines 13-30 should match the tags in the newEvent.js file.

File: views/newEvent.handlebars [hunk 4]
```diff
@@ -1,112 +1,142 @@
...
+                  <option value="VA">Virginia</option>
+                  <option value="VT">Vermont</option>
+                  <option value="WA">Washington</option>
+                  <option value="WI">Wisconsin</option>
+                  <option value="WV">West Virginia</option>
+                  <option value="WY">Wyoming</option> 
+                </select>
+              </div>
+              <div class="form-group col-md-2">
+                <label for="inputZip">Zip</label>
```
Comment (ddaghlas) [inline views/newEvent.handlebars:89]: We need to create a function to handle the zip input in newEvent.js

File: views/newEvent.handlebars [hunk 5]
```diff
@@ -1,112 +1,142 @@
...
+    <!-- Add event tasks -->
+    <div class="row mb-3 justify-content-center">
+      <h2>Add Volunteer Tasks</h2>
+    </div>
+
+    <form class="justify-content-center d-flex mb-3">
+      <div class="input-group col-md-6 mb-3">
+        <input type="text" class="form-control" placeholder="Add volunteer task..." aria-label="Volunteer task" aria-describedby="button-addon2">
+        <div class="input-group-append">
+          <button class="btn btn-outline-light" type="button" id="button-addon2">Add Task</button>
```
Comment (ddaghlas) [inline views/newEvent.handlebars:123]: id should be added to match the addTask.js

**Review by ddaghlas — COMMENTED** [review]
Added some comments for further review.

File: public/js/datepicker.js (hunk 1 above)
Comment (danshoup) [reply public/js/datepicker.js:3]: Yep.  Gonna totally change the file

File: views/newEvent.handlebars (hunk 3 above)
Comment (danshoup) [reply views/newEvent.handlebars:30]: Next step is to check the routes and make sure tags are appropriate

File: views/newEvent.handlebars (hunk 5 above)
Comment (danshoup) [reply views/newEvent.handlebars:123]: Note to match IDs

**Review by ddaghlas — APPROVED** [review]
Looks good so far.

**Pull request merged**